main.lua

```diff
@@ -152,13 +152,13 @@ local function build_status_response()
   fail = false
   for k, v in pairs(values) do
     if v == '' then
-      mp.msg.log("WARN", 'Could not fetch "'.. k .. '" from mpv.')
+      mp.msg.debug('Could not fetch "'.. k .. '" from mpv.')
```

Comment:
Altough I don't necessarily disagree with this change, it has nothing to do with the feature this commit implements.